internal/parse/parser_internal_test.go

```diff
@@ -29,11 +29,16 @@ func TestRunTable(t *testing.T) {
 		{bytef: p.skipByteFind, result: []bool{false, true, true}, input: "abcde", data: []string{"x", "b", "c"}},
 		{bytef: p.skipByteFind, result: []bool{true, false}, input: "abcde ", data: []string{" ", " "}},
 
-		{stringf0: p.skipSpaces, result: []bool{false}, input: "", data: []string{}},
-		{stringf0: p.skipSpaces, result: []bool{false}, input: "abc    d", data: []string{}},
-		{stringf0: p.skipSpaces, result: []bool{true}, input: "     abcd", data: []string{}},
-		{stringf0: p.skipSpaces, result: []bool{true}, input: "  \t  abcd", data: []string{}},
-		{stringf0: p.skipSpaces, result: []bool{false}, input: "\t  abcd", data: []string{}},
+		{stringf0: p.skipBlanks, result: []bool{false}, input: "", data: []string{}},
+		{stringf0: p.skipBlanks, result: []bool{false}, input: "abc    d", data: []string{}},
```

Comment:
Would it be relevant to have test cases for `\n \r \t` also mixed up in the whitespaces as in here:
`input: "abc  \t  d"`
Just used \t here, but maybe others as well?

Reply:
Those should be covered with the tests around line 37 and such.